app/src/main/java/chat/rocket/android/helper/OkHttpHelper.java

```diff
@@ -1,21 +1,36 @@
 package chat.rocket.android.helper;
 
 import android.content.Context;
+import android.support.annotation.NonNull;
 import com.facebook.stetho.okhttp3.StethoInterceptor;
 
+import java.io.IOException;
 import java.util.concurrent.TimeUnit;
 import chat.rocket.android.RocketChatCache;
 import chat.rocket.android.api.rest.CookieInterceptor;
 import chat.rocket.android.api.rest.DefaultCookieProvider;
+import okhttp3.Call;
+import okhttp3.Callback;
 import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
 
 /**
  * Helper class for OkHttp client.
  */
 public class OkHttpHelper {
+  private static OkHttpClient defaultHttpClient;
   private static OkHttpClient httpClientForUploadFile;
   private static OkHttpClient httpClientForDownloadFile;
   private static OkHttpClient httpClientForWS;
+  private static String contentType;
```

Comment:
This will not work as expected...

First time you call `getContentType()` it will probably return `null`, since it is a async request.
Next time you call it, you will probably get the contentType from the previous request, which may be right or not.

On a list flinging, there will be dozens of requests overwriting `contentType`